base/sparse/sparsevector.jl

```diff
@@ -646,10 +646,11 @@ function getindex(A::SparseMatrixCSC{Tv}, I::AbstractVector) where Tv
     colptrA = A.colptr
     rowvalA = A.rowval
     nzvalA = A.nzval
+    Ti = eltype(colptrA)
```

Comment:
Same as above; I suggest we update the function signature instead;
```julia
function getindex(A::SparseMatrixCSC{Tv,Ti}, I::AbstractVector) where {Tv,Ti}
    # ...
```

Reply:
Changed.